.devcontainer/Dockerfile

```diff
@@ -1,7 +1,15 @@
-FROM gnuoctave/octave:7.2.0
+FROM gnuoctave/octave:8.1.0
 
-RUN octave --eval 'pkg install -forge statistics'
-RUN apt update && apt install -yq libnetcdf-dev && octave --eval 'pkg install -forge netcdf'
+# NOTE: all package version are pinned to avoid the container breaking with the 
+#     latest version on octave's forge.
+#
+# IO is a dependency of statistics, needs to be installed first.
+RUN octave --eval 'pkg install "https://downloads.sourceforge.net/project/octave/Octave%20Forge%20Packages/Individual%20Package%20Releases/io-2.6.4.tar.gz"'
+RUN octave --eval 'pkg install "https://octave.sourceforge.io/download.php?package=statistics-1.4.3.tar.gz"'
+
+# Install netcdf dependencies first.
+RUN apt update && apt install -yq libnetcdf-dev
+RUN octave --eval 'pkg install "https://downloads.sourceforge.net/project/octave/Octave%20Forge%20Packages/Individual%20Package%20Releases/netcdf-1.0.16.tar.gz"'
```

Comment:
The netcdf package was needed for `ncread` function. However, after #143, we don't need the package anymore. see my [comment](https://github.com/EcoExtreML/STEMMUS_SCOPE/pull/143#pullrequestreview-1287577069).

Reply:
Oh yeah, I'll remove the netcdf installation.